docs/ref/models/fields.txt

```diff
@@ -373,6 +373,33 @@ this field's index, if this field is indexed. The default is the project's
 :attr:`~Options.db_tablespace` of the model, if any. If the backend doesn't
 support tablespaces for indexes, this option is ignored.
 
+``db_default``
+--------------
+
+.. versionadded:: 5.0
+
+.. attribute:: Field.db_default
+
+The default value for the field in the database. This can be a literal value or
+a database function, such as :class:`~django.db.models.functions.Now`::
+
+    created = DateTimeField(db_default=Now())
+
+More complex expressions can be used, as long as they are made from literals
+and database functions::
+
+    month_due = DateField(db_default=TruncMonth(Now() + timedelta(days=90)))
+
+However, a database default may not reference another field or model. This is
+invalid::
+
+    end = IntegerField(db_default=F("start") + 50)
+
+If both ``db_default`` and :attr:`Field.default` are set, ``default`` will take
+precedence in python code. ``db_default`` will still be set at the database
+level, so raw SQL will use this value. ``db_default`` will also be used when
```

Comment:
I'd chop `so raw SQL will use this value`, IMO, it's unnecessary and a bit confusing.